website/src/pages/en/token-api/faq.mdx

```diff
@@ -86,20 +94,24 @@ Networks that are currently or temporarily unavailable on a given endpoint will
 
 ### Are there rate limits or usage costs?
 
-During Beta, the Token API is free for authorized developers. There are no specific rate limits, but reasonable throttling exists to prevent abuse. High request volumes may trigger HTTP 429 errors. Monitor official announcements for future pricing changes after Beta.
+During Beta, the Token API has a generous allowance of free usage. There are no specific rate limits, but reasonable throttling exists to prevent abuse. High request volumes may trigger HTTP 429 errors. Monitor official announcements for future pricing changes after Beta.
 
 ### What do I do if I notice data inconsistencies in the data returned by the Token API?
 
 If you notice data inconsistencies, please report the issue on our [Discord](https://discord.gg/graphprotocol). Identifying edge cases can help make sure all data is accurate and up-to-date.
 
 ### How do I specify a network?
 
-You can query available networks with [this link](https://token-api.thegraph.com/networks). A full list of the exact network IDs accepted by The Graph can be found [here](/supported-networks/). Use the optional `network_id` parameter (e.g., `mainnet`, `bsc`, `base`, `arbitrum-one`, `optimism`, `matic`, `unichain`) to target a specific chain. Without this parameter, the API defaults to Ethereum mainnet.
+You can query which networks are available with [this link](https://token-api.thegraph.com/networks). Take note that the network IDs differ between the Token API and Subgraphs. For EVM networks, use EVM endpoints with the optional `network_id` parameter (e.g., `mainnet`, `bsc`, `base`, `arbitrum-one`, `optimism`, `polygon`, `unichain`) to target a specific chain. Without this parameter, the API defaults to Ethereum mainnet.
+
+To query Solana, use the SVM endpoints.
 
 ### Why do I only see 10 results? How can I get more data?
 
 Endpoints cap output at 10 items by default. Use pagination parameters: `limit` and `page` (1-indexed) to return more results. For example, set `limit=50` to get 50 results, and increment `page` for subsequent batches (e.g., `page=2` for items 51-100).
 
+It is currently possible for the total number of pages to be inexact. This is because the ClickHouse backend doesn't read the whole table, and instead provides an approximation of the number of records, in order to make queries respond faster. To avoid getting confused by this approximation, make sure to continue fetching next pages until there is no more data, or the endpoint returns an error.
```

Comment:
Is "the ClickHouse backend" really relevant to consumers?

Reply:
I think that would be better!

Reply:
```suggestion
It is currently possible for the total number of pages to be inexact. This is because the backend provides only an approximation of the number of records in order to improve response time. To avoid getting confused by this approximation, make sure to continue fetching next pages until there is no more data, or the endpoint returns an error.
```